.circleci/real_config.yml

```diff
@@ -675,13 +675,16 @@ commands:
 
             # Remove prior test results
             rm -f "<<parameters.junit-path>>"
+            echo "Removed prior results"
 
             pytest --setup-plan -m '<<parameters.mark>>' | egrep '\.py s*$' | sed 's/\.py.*$/.py/' > /tmp/all-relevant-files
+            echo "Relevant files:"
+            cat /tmp/all-relevant-files
             if [ ! -s /tmp/all-relevant-files ]; then
               echo 'No test files found!'
               exit 1
             fi
-
+            
```

Comment:
Bonus spaces!

Reply:
accidental +1 LLOC change 😜 I'll remove it